pkg/protocol/http.go

```diff
@@ -132,7 +136,7 @@ func (nr *NetHTTPRequest) StartRequest() {
 	}
 	var span opentracing.Span
 	if err != nil {
-		log.Printf("Carrier extract error: %s", err.Error())
+		nr.logger.Errorf("Carrier extract error: %s", err.Error())
```

Comment:
it will be ok in most cases, because we can be the first point in tracing path and don't have uber-ctx header. I think this is info level.